src/ui/components/NewProject/index.jsx

```diff
@@ -1,5 +1,189 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+
+import Button from "../shared/Button";
+import ToastPopup from "../shared/Toast";
+
+import useProjectStore from "../../../store/project";
+import useProjectVersionStore from "../../../store/projectVersion";
+import getProjectKeyFromURI from "../../../utils/getProjectKeyfromURI";
+import getAllVersions from "../../../service/getAllVersions";
+import isValidFigmaUrl from "../../../utils/isValidFigmaUrl";
+import postMessage from "../../../utils/postMessage";
+
 function NewProject() {
-  return <h1>NewProject</h1>;
+  const navigate = useNavigate();
+
+  const { project, setProject, clearProject } = useProjectStore();
+  const { setVersion, clearVersion } = useProjectVersionStore();
+
+  const [inputValue, setInputValue] = useState(null);
+  const [toast, setToast] = useState(false);
+
+  useEffect(() => {
+    clearProject();
+    clearVersion();
+  }, []);
+
+  const handleChangeInput = ev => {
+    setInputValue(ev.target.value);
+  };
```

Comment:
- `P2`
`inputValue`를 따로 선언하지 않고
```suggestion
  const handleChangeInput = ev => {
    setProject({ projectUrl: ev.target.value });
  };
```
이렇게 작성하는것도 가능하지 않을까요?

```js
useEffect(() => {
    window.addEventListener("message", handleMessage);

    return () => {
      window.removeEventListener("message", handleMessage);
    };
  }, [project.projectUrl]);
```

혹시 제한되는 부분이 있는지 궁금합니다!

Reply:
지역 상태로 관리하는 게 주니 좋네요! 테스트했을 때 문제 없이 버전 선택까지 넘어가네요 :)

글로벌 상태를 바로 쓸 생각을 못했는데 예리하게 짚어주셔서 감사합니다!
반영해두겠습니다! 👍